packages/@react-aria/textfield/src/useTextField.ts

```diff
@@ -79,7 +79,12 @@ export interface AriaTextFieldOptions<T extends TextFieldIntrinsicElements> exte
    * For example, [`type`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-type).
    * @default 'input'
    */
-  inputElementType?: T
+  inputElementType?: T,
+  /**
+   * A nonstandard attribute used by iOS Safari that controls how textual form elements should be automatically capitalized.
+   * See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#autocapitalize).
```

Comment:
i did notice this link is wrong, should go here
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocapitalize

which I guess would lead this to accepting the type
`'none' | 'sentences' | 'words' | 'characters'`
instead of what form accepted

Reply:
@snowystinger 
The link was wrong and will be corrected.
However, regarding type, according to the [HTML Standard](https://html.spec.whatwg.org/multipage/interaction.html#autocapitalization), I feel it is correct to accept on and off as keywords as well, what do you think?

<img width="255" alt="image" src="https://github.com/adobe/react-spectrum/assets/76232929/9aab1420-ee50-420e-b632-b5e20629fb20">

Reply:
It's probably worth it to do some testing to see which ones are actually supported. The spec is a good guideline, and if we find gaps in coverage, we perhaps could map the missing values.
Would you mind getting us started on that?

Reply:
Sure thing, I'll get started on the testing and report back with the results.

Reply:
@snowystinger 
I tested autocapitalize on iOS Safari 16.6 using the link below and found that the 'off' setting behaves identically to 'none', and 'on' mirrors 'sentences'.
https://codepen.io/ryo-manba/pen/rNPrwWM

Based on these findings, it appears that our current PR approach is accurate.

Reply:
Thanks! I checked iOS 17.1, and my coworker checked on an Android with the same results
appears Mozilla might be a little outdated on usage